integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -40,7 +40,7 @@
 import tachyon.heartbeat.HeartbeatScheduler;
 import tachyon.job.CommandLineJob;
 import tachyon.job.JobConf;
-import tachyon.master.lineage.meta.LineageFileState;
+import tachyon.proto.JournalEntryProtos.LineageFileState;
```

Comment:
I have been doing the latter for Thrift option structs (i.e. there is a Thrift struct and Java struct that we convert back and forth between)